### ISSUE TITLE
getInitialProps called twice only when using chrome

### ISSUE THREAD

**Issue by mzygmunt** [zeit/next.js#9056]
# Bug report

## Describe the bug

getInitialProps in Index page is called twice when using chrome, but when using safari works as expected. This may have something to do with loading favicon.

## To Reproduce

1. Clone https://github.com/mzygmunt/getinitialprops-run-twice-on-chrome
2. Run: npm install, npm start dev
2. Open http://localhost:3000 using chrome
3. See twice text "FOOBAR" in server console on every page refresh

## Expected behavior

After refreshing the page/or initial load, the getInitialProps function calls only once as in safari.

## Screenshots

![image](https://user-images.githubusercontent.com/13297558/66707056-5f588180-ed3b-11e9-88df-6611bcfb9643.png)

## System information

- OS: macOS 10.15
- Browser: Chrome 77.0.3865.90 
- Version of Next.js: 9.1.1

**Comment by timneutkens** [comment]
I highly doubt this is the case but someone can look into it if they want to.

**Comment by giuseppeg** [comment]
I doubt it is the case too, unless the app is more convoluted than the test case. @mzygmunt I got two consecutive `FOOBAR` in the console too but the first is from the previous request. On refresh the new request prints only one. Maybe you have a more deterministic way to reproduce?

**Comment by mzygmunt** [comment]
I found a solution. One of the chrome extensions forced the request to re-execution. Sorry for the hassle.

**Comment by jonkwheeler** [comment]
@mzygmunt what was the Chrome extension?

**Comment by mzygmunt** [comment]
@jonkwheeler In my case was Adblock Plus.

**Comment by jonkwheeler** [comment]
Have to throw this out there. Why would an Ad Blocker make `getInitialProps` fire twice? That would be not favorable. Any ideas?

**Comment by srosset81** [comment]
Apparently AdBlock Plus fetch the page twice, which is why `getInitialProps` is called twice.

**Comment by jonkwheeler** [comment]
I'm using `uBlock Origin`. Regardless, it's not favorable this would happen twice, as lots of people use Ad Blockers of some kind.

**Comment by timneutkens** [comment]
I think that @srosset81 means it's not related to Next.js, just that adblocker literally `fetch()`ing the page to verify the source.

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
